shell/packages/sandstorm-ui-app-details/package.js

```diff
@@ -0,0 +1,28 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+Package.describe({
+  summary: "Sandstorm UI app details page",
+  version: "0.1.0"
+});
+
+Package.onUse(function (api) {
+  api.use(["check", "reactive-var", "reload", "templating", "tracker", "underscore", "sandstorm-identicons"], "client");
+  api.addFiles(["app-details-common.js"], ["client","server"]);
+  api.addFiles(["app-details.html", "app-details-client.js"], "client");
+  api.addFiles(["app-details-server.js"], "server")
```

Comment:
Since the server file is empty, and -common.js contains code only used on the client, I'd go ahead and make this a client-only package. I think that's probably the right thing for all our UI packages, really.